server/ERNI.PBA.Server.Host/Model/RequestInputModel.cs

```diff
@@ -1,13 +1,13 @@
+﻿using System;
+
 namespace ERNI.PBA.Server.Host.Model
 {
-    public class PostRequestModel
```

Comment:
I'd rather get rid of any inheritance in models, even f it means code duplication